server/block/register.go

```diff
@@ -473,6 +476,9 @@ func init() {
 			world.RegisterItem(Copper{Type: c, Oxidation: o, Waxed: true})
 		}
 	}
+	for c := range 5 {
+		world.RegisterItem(RespawnAnchor{Charge: c})
```

Comment:
We typically do not register items that are never accessible in the vanilla game. Only a RespawnAnchor with Charge 0 should be registered.